### DISCUSSION TITLE
I'm running into a parse error on line 213

### DISCUSSION THREAD

**Discussion by garryweil** [FuLoRi/Analyze-LastPassVaultGUI#20] [Q&A]
I had no problem running the non-GUI version 1.1; error message:
ParserError: C:\Users\gweil1x\OneDrive\Documents\LastPass\Analyze-LastPass-Vault\Analyze-LastPassVaultGUIv1.2.ps1:213
Line |
 213 |  … k Button--medium Button d-lg-none color-fg-inherit p-1">    <span cla …
     |                                                                                                            ~
     | The '<' operator is reserved for future use.

I thought it was my version of PS, so I ugraded from 7.9.2 to 7.3.3 but I still get this error. 
Can anyone help??

**Comment by FuLoRi** [comment]
Apologies for the delayed response. Hopefully, you've figured it out by now. It looks like the script file you were running is corrupted. I would suggest you try downloading it again and then opening it in Notepad to make sure it matches what you see [here](https://github.com/FuLoRi/Analyze-LastPassVaultGUI/blob/main/Analyze-LastPassVaultGUI.ps1) before running it.
